src/main.rs

```diff
@@ -424,6 +425,44 @@ fn set_trace_id() {
   std::env::set_var(key, my_id)
 }
 
+fn cmd_info(
+    _config: Arc<Config>,
+    target_path: PathBuf,
+    _diff: bool,
+    _overview: bool,
+    _branch: bool,
+    _local: bool
+) -> Result<i32, Error> {
+    let manifest_dir = target_path.join(".repo").join("manifests");
+    let manifest_path = target_path.join(".repo").join("manifest.xml");
+    let manifest = Manifest::parse(&manifest_dir, &manifest_path)?;
+    let manifest_default = manifest.default.expect("manifest missing <default>");
+    let manifest_default_revision = manifest_default.revision.expect("manifest <default> missing revision");
+
+    // Look up manifest git project branch information via git config
+    let manifests_git_config_path = manifest_dir.join(".git").join("config");
+    let manifests_git_config_path = Path::new(&manifests_git_config_path);
+    let manifest_git_config = git2::Config::open(manifests_git_config_path).expect("unable to open manifests git config");
+
+    let default_merge_branch = manifest_git_config.get_str("branch.default.merge").unwrap_or("refs/heads/master");
```

Comment:
I don't think we should be opening the git project for this, we can just look at `Tree::branch`.